src/js/components/help/__snapshots__/devices.test.js.snap

```diff
@@ -1,66 +1,141 @@
-    target="_blank"
-  >
-    Read the API documentation for more on static groups and their creation
-  </a>
-  .
-  <br />
-  <h3>
-    Dynamic groups
-  </h3>
-  Unlike static groups, dynamic group membership is defined by matching a set of filters on the device inventory attributes. Since these attribute values might change over time, a device might match different groups because of changing attribute values, thus leading to the group changing dynamically.
-  <a
-    href="https://docs.mender.io/apis/enterprise/management-apis/device-inventory-v2#filters-post"
-    target="_blank"
-  >
-    Read the API documentation for more information on the creation of dynamic groups
-  </a>
-  .
-  <p
-    className="info icon"
+<div
+  className="flexbox column"
+>
+  <div
+    className="one-column margin-bottom"
+    style={
+      Object {
+        "flexGrow": 1,
+        "padding": 0,
+      }
+    }
   >
-    <svg
-      aria-hidden={true}
-      className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
-      focusable="false"
+    <div
+      className="margin-right-large"
+    >
+      <div
+        className="flexbox margin-bottom"
+        style={
+          Object {
+            "alignItems": "center",
+          }
+        }
+      >
+        <svg
+          aria-hidden={true}
+          className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
+          focusable="false"
+          style={
+            Object {
+              "color": "#7b7b7b",
+            }
+          }
+          viewBox="0 0 24 24"
+        >
+          <path
+            d="M18 8h-1V6c0-2.76-2.24-5-5-5S7 3.24 7 6v2H6c-1.1 0-2 .9-2 2v10c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V10c0-1.1-.9-2-2-2zM9 6c0-1.66 1.34-3 3-3s3 1.34 3 3v2H9V6zm9 14H6V10h12v10zm-6-3c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"
+          />
+        </svg>
+        <div
+          className="bold margin-left-small"
+        >
+          Static group
+        </div>
+      </div>
+      <p
+        className="help-content"
+      >
+        Select specific devices to add them to a static group. Filter to find devices in the 'All devices' UI, check the devices you want to add, and click the action button to assign them to a group.
+      </p>
+    </div>
+    <img
+      src="static-group-creation.gif"
       style={
         Object {
-          "margin": "0 6px 4px 0",
-          "verticalAlign": "middle",
+          "maxWidth": "100%",
         }
       }
-      viewBox="0 0 24 24"
-    >
-      <path
-        d="M11 7h2v2h-2zm0 4h2v6h-2zm1-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"
-      />
-    </svg>
-    With a Mender Enterprise plan you can save dynamic groups and ease device management. 
-    <a
-      href="https://mender.io"
-      target="_blank"
+    />
+  </div>
+  <div
+    className="one-column lightestgrey"
+    style={
+      Object {
+        "flexGrow": 1,
+        "padding": 0,
```

Comment:
The only thing I'd suggest is to add 30px padding here to the lightestgrey section, just for aesthetic reasons

Reply:
Done!